packages/Color/src/Color.tsx

```diff
@@ -33,11 +35,19 @@ const Color: React.FunctionComponent<ColorProps> = ({
     className,
     color,
     dataTest,
+    isSquare = false,
```

Comment:
A color is round by default with the workleap theme.
A color is square by default with the igloo theme.

There is a props isSquare that is false by default, and probably do nothing in the igloo theme?
should this default value be ajusted by the current brand? and isSquare=false with the igloo theme should be round?

Reply:
I'm a bit confused.. but yes it does nothing for the Igloo brand.
don't think we can edit defaults based on brand. 
It would be pointless to do it after since it wouldn't be shown in the storybook props table.
The igloo theme doesn't have a round Color. it's always square.